CorsixTH/Lua/humanoid_actions/seek_reception.lua

```diff
@@ -96,7 +96,7 @@ local function action_seek_reception_start(action, humanoid)
       end
     end
   local desks = #humanoid.hospital:findReceptionDesks()
-  local receptionists = humanoid.hospital:countStaffOfCategory("Receptionist")
+  local receptionists = humanoid.hospital:countStaffOfCategory("Receptionist", 1)
   if (receptionists > 1 and desks > 0) or (receptionists > 0 and desks > 1) then
     local queueavg = math.floor(queuetotal / desks)
     if receptionists < desks and queueavg > 5 then
```

Comment:
`receptionists < desks` changes too, better just compute the actual number of receptionists I guess